.circleci/config.yml

```diff
@@ -173,7 +173,9 @@ jobs:
             demisto-sdk create -a $CIRCLE_ARTIFACTS
             # create zip with contents of content_new.zip and content_test.zip for use in updating content on instances
             cp "$CIRCLE_ARTIFACTS/content_new.zip" "$CIRCLE_ARTIFACTS/all_content.zip"
-            zip -j "$CIRCLE_ARTIFACTS/all_content.zip" "$CIRCLE_ARTIFACTS/bundle_test/*"
+            unzip -q "$CIRCLE_ARTIFACTS/content_test.zip" -d "$CIRCLE_ARTIFACTS/test_content"
```

Comment:
change the destination of the temp folder (use CIRCLE_ARTIFACTS only for actual artifacts).
do it in another PR.

Reply:
what does it matter if I’m deleting it in the same build step anyway?

Reply:
if step crashed before it removed the folder, it will be posted as an artifact.
don't know what are the effects for so many files as artifacts and i don't want to find out...

Reply:
legit. It shall be as you say sensei. 
![image](https://user-images.githubusercontent.com/46294017/71217172-7d0b0f80-22c5-11ea-8edc-f3e3c7d93e0e.png)